styles/base/form.scss

```diff
@@ -93,28 +80,25 @@ label {
     z-index: 2;
     float: left;
     width: 100%;
+    border-right: 0;
+    border-left: 0;
+
+    &:focus { 
+      box-shadow: 0 0 2px 1px rgba(82,181,229,1);
```

Comment:
i just base our input on `primercss`'s and `bootstrap`'s input. i thought it might be good to add box-shadow instead of border